src/Common/src/System/LocalAppContext.cs

```diff
@@ -40,6 +38,17 @@ private static bool GetCachedSwitchValueInternal(string switchName, ref int swit
             return isSwitchEnabled;
         }
 
+        private static void InitDisableCaching()
+        {
+            bool isEnabled;
+            if (AppContext.TryGetSwitch(@"TestSwitch.LocalAppContext.DisableCaching", out isEnabled))
```

Comment:
Ok I already have that implementation, just wanted to make sure I got your idea :) will update the PR! Thanks